site/content/contribute/getting-started/code-reviews.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Code Reviews"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+All changes to the product must be reviewed.
+
+* User experience changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+* Code changes must be reviewed by at least two [core committers](http://localhost:1313/contribute/getting-started/core-committers/#core-committers).
+* Documentation changes should be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers) and at least one [core committer](/contribute/getting-started/core-committers/#core-committers).
+
+If you are not a core committer
+-------------------------------
+
+1. Submit your pull request.
+2. Reach out to the author of the ticket your PR resolves.
+    * Typically this means [mentioning](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#mentioning-people-and-teams) their username as a comment on the pull request.
+    * Product managers and core committers are often on the lookout for new PRs, and may handle this for you automatically.
+    * When in doubt, ask for help in the [Developers](https://community.mattermost.com/core/channels/developers) channel on our community server.
+3. Wait for a review.
+    * The ticket author, another core committer or a product manager will assign reviewers and label your pull request appropriately.
+    * Expect some interaction with at least one reviewer within 5 business days.
+    * Keep in mind that Mattermost Core Committers are geographically distributed around the world and likely in a different time zone than your own.
+4. Make any necessary changes.
+    * If a reviewer requests changes, your pull request will disappear from their queue of reviews.
+    * Once you've addressed the concerns, reach out with another mention or via our community server.
+5. Wait for your code to be merged.
+    * Once all reviewers have approved your changes, they will handle merging your code.
+
+If you are a core committer seeking a review
+--------------------------------------------
+
+1. Submit your pull request.
+2. Assign a [product manager](/contribute/getting-started/core-committers/#product-managers) to your review and label your pull request with `1: PM Review`.
+    * If your changes do not affect the user experience, you may skip this step.
+    * Wait for their review before continuing, to avoid churn if changes are requested.
+3. Assign two [core committers](/contribute/getting-started/core-committers/) to your review and label your pull request with `2: Dev Review`.
```

Comment:
Should PMs stop assigning dev reviews? (I've been doing so after I've approved PM review)

Reply:
0/5 on whether PMs should assign reviews when they are involved in a `PM Review`, or if we should defer to the core committers knowledge of expertise as such. How do you usually pick core committers, @jasonblais?

Reply:
One member from the team the Jira ticket is assigned to, and a second reviewer based on intuition (who I think would be the best reviewer)

Reply:
I'll add a note to the step prior as such.